### PR TITLE
added nodejs week2 homework

### PR DESCRIPTION
**Pull request by DeepthiDukka**

### CODE REVIEW

File: nodejs/week2/warm_up/index.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+//Lets use express routes to create a small calculator:
+
+const express = require("express");
+const app = express();
+const port = 3000;
+
+app.get('/numbers/add', (req, res) => {
+    let sum = Number(req.query["first"]) + Number(req.query["second"]);
```
Comment (OlkaA) [inline nodejs/week2/warm_up/index.js]: I think` req.query["first"]` is overcomplicated.
You can use just `req.query.first` and get the same value.

File: nodejs/week2/meal_sharing_website/src/backend/routes/meals.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+//Get meals that has a price smaller than maxPrice
+router.get("/", (req, res) => {
+	let meal;
+	const maxPrice = parseFloat(req.query.maxPrice);
+	const title = req.query.title;
+	const date = parseInt(req.query.createdAfter);
+	const limit = parseInt(req.query.limit);
+
+	if (maxPrice >= 0) {
```
Comment (OlkaA) [inline nodejs/week2/meal_sharing_website/src/backend/routes/meals.js]: It is not necessity to compare maxPrice with 0. You can just check if it is exists.
Like i`f (maxPrice) {`

the same for limit.

**Review by OlkaA — APPROVED** [review]
Good job with warmup and meal sharing 👍
I have added some small comments to the PR.

File: nodejs/week2/warm_up/index.js (hunk 1 above)
Comment (DeepthiDukka) [reply nodejs/week2/warm_up/index.js]: Implemented the given feedback

**Pull request merged**